foxio/package.yaml

```diff
@@ -168,35 +168,41 @@ examples:
       package](https://github.com/FoxIO-LLC/ja4/tree/main/zeek) to enrich your
       Zeek logs with JA4+ fingerprints.
     definition: |
-      subscribe zeek
-      | where #schema == "zeek.conn"
-      | enrich ja4t --field=ja4t
-      | enrich ja4ts --field=ja4ts
+      // tql2
+      subscribe "zeek"
+      where @name == "zeek.conn"
+      context::enrich "ja4t", key=ja4t
+      context::enrich "ja4ts", key=ja4ts
 
   - name: FoxIO database top-10 operating systems
     description: |
       Shows a bar chart of the top-10 most common operating systems in the JA4+
       database.
     definition: |
-      from "https://ja4db.com/api/read/" read json --arrays-of-objects
-      | where os != null
-      | top os
-      | head
-      | chart bar
+      // tql2
+      load_http "https://ja4db.com/api/read/"
+      read_json arrays_of_objects=true
```

Comment:
I think we replace the `load`/`read` combos with `from` where possible.

Reply:
You can't pass options for `read` through `from`. It will still need to be two different steps.

Reply:
Ah, gotcha.